src/xenia/base/string_util.h

```diff
@@ -54,8 +54,8 @@ inline std::string to_hex_string(double value) {
 
 inline std::string to_hex_string(const vec128_t& value) {
   char buffer[128];
-  std::snprintf(buffer, sizeof(buffer), "[%.8X, %.8X, %.8X, %.8X]",
-                value.u32[0], value.u32[1], value.u32[2], value.u32[3]);
+  snprintf(buffer, sizeof(buffer), "[%.8X, %.8X, %.8X, %.8X]", value.u32[0],
```

Comment:
I don't think this belongs to this PR. Why did you drop `std::`?

Reply:
Clarified the drop of std:: in the commit message, copying here:
This seems to be an issue with Microsoft declaring a snprintf macro in an included header which causes issues.
Related issues:

nlohmann/json#1408
kodi-pvr/pvr.vuplus#162